pkgs/games/sm64ex/default.nix

```diff
@@ -25,14 +25,14 @@
 }:
 
 stdenv.mkDerivation rec {
-  pname = "sm64ex";
-  version = "unstable-2020-06-19";
+  pname = "sm64ex-unstable";
+  version = "2020-10-09";
```

Comment:
```suggestion
  pname = "sm64ex";
  version = "unstable-2020-10-09";
```

Reply:
See https://github.com/NixOS/nixpkgs/issues/68518, it's supposed to be like that

Reply:
This only applies if we have two versions of the same package.

Reply:
What are you basing this off of? I cannot seem to find any documentation backing that up.

Reply:
Any of the 3 PRs and issues that is about this which I will not find because github search is not great. This is basically jtojnar suggestion to not introduce new packages to repology and keep nix env working.

Reply:
Apologies for the long wait, totally forgot about this one.

Fixed this now :)